content-node/docker-compose/auto-upgrade.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
this is an adapted version of the files on the testnet nodes, main difference is the paths are not hardcoded and there's a diff to restart docker so it doesn't restart every time cron runs